cni/cmd/tc-redirect-tap/main.go

```diff
@@ -229,9 +274,106 @@ func (p plugin) add(currentResult *current.Result) error {
 }
 
 func (p plugin) del() error {
-	panic("del is currently unimplemented")
+	return p.netNS.Do(func(_ ns.NetNS) error {
+		var multiErr *multierror.Error
+
+		redirectLink, err := p.GetLink(p.redirectInterfaceName)
+		switch err.(type) {
+		case *internal.LinkNotFoundError:
+			// if the link doesn't exist, there's nothing to do
+		default:
+			if err != nil {
```

Comment:
`case nil:`? Maybe personal preference though.

Reply:
Oh nice I didn't know you could have a `case nil` statement, updated with that